src/src/Helpers/ESPEasy_FactoryDefault.cpp

```diff
@@ -216,6 +216,8 @@ void ResetFactory()
    */
   Settings.I2C_clockSpeed = DEFAULT_I2C_CLOCK_SPEED;
 
+  Settings.JSONBoolWithQuotes(DEFAULT_JSON_USE_QUOTES);
```

Comment:
There is a function to output a "JSONBool" string.
That would be a good start for finding uses.
But maybe just have a look at where we decide whether it is a numerical or not, thus wrapping quotes around its value.
Then you have it all I guess.

Reply:
I think I've got them covered, both for the real JSON output and the `addHtml` variant.